Gemfile

```diff
@@ -1,8 +1,9 @@
 source 'https://rubygems.org'
 
-ruby '3.2.2'
-gem 'puma', '6.4.2'
+ruby File.read(".ruby-version").strip
+
 gem 'rails', '~> 7.1.3'
+gem 'puma', '6.4.2'
```

Comment:
Unrelated to this feature, but a nice value add for reducing the hardcoded ruby `3.2.2` instances in the app. Also I moved the rails gem up so it is easier to spot right away.